testsupport/init.go

```diff
@@ -263,3 +276,10 @@ func schemeWithAllAPIs(t *testing.T) *runtime.Scheme {
 	require.NoError(t, builder.AddToScheme(s))
 	return s
 }
+
+func IsSecondMemberMode(t *testing.T) bool {
+	secondMemberMode := os.Getenv(wait.SecondMemberModeVar)
```

Comment:
Are you sure that this variable is available at the time of the test execution? It is not explicitly set in the makefile https://github.com/codeready-toolchain/toolchain-e2e/blob/0c385c92fd63315225dde4092f46be0479ef296d/make/test.mk#L167
What is set is the `MEMBER_NS_2` that is already retrieved in the test suite, so you could technically rely on that one - if it's empty then it's running in a "single member mode"

Reply:
I forgot to update there! Sorry, thanks!

I thought about using `MEMBER_NS_2`, but then I saw that we have `SECOND_MEMBER_MODE`, and I decided to use that one.  Essentially because I think it would be more intuitive if we point `SECOND_MEMBER_MODE=false`, when using the make targets like `make prepare-e2e deploy-e2e SECOND_MEMBER_MODE=false (...)` in WA

Reply:
either option is fine :-)